pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -57,20 +96,21 @@ type RoleModificationOptions struct {
 	DryRun bool
 	Output string
 
-	PrintObj func(obj runtime.Object) error
+	PrintObj  func(obj runtime.Object) error
+	PrintErrf func(format string, args ...interface{})
 }
 
 // NewCmdAddRoleToGroup implements the OpenShift cli add-role-to-group command
-func NewCmdAddRoleToGroup(name, fullName string, f kcmdutil.Factory, out io.Writer) *cobra.Command {
+func NewCmdAddRoleToGroup(name, fullName string, f kcmdutil.Factory, out, errOut io.Writer) *cobra.Command {
```

Comment:
@juanvallejo @soltysh looks like a package was missed.  I'm breaking out my "I told you so" here.